### DISCUSSION TITLE
Multiple plan options for the same EIN

### DISCUSSION THREAD

**Discussion by todd-kooser** [CMSgov/price-transparency-guide#174] [Q&A]
We have many employer groups that offer multiple plan options to their employees. These plans do not have a HIOS ID assigned to them, so presumably the plan ID would be the employer's EIN, however that would result in multiple plans with the same EIN. Is it intended that there be multiple plans with the same plan ID, or is there another way to differentiate these plans?

**Comment by ghost** [comment]
@todd-kooser I have seen the same scenario in the case of large and/or self funded employer groups that would provide multiple tailored plans for their employees usually with the differential in the plans revolving around the member benefits of the plan and not the provider negotiated rates. Because of the File Naming Convention being <YYYY-MM-DD>_<payer or issuer name>_<plan name>_<file type name>.<file extension>, you would end up creating a separate file per plan name where in the root schema you would have duplicate plan_id _type, plan_id and plan_market_type fields if the plans all shared the same EIN.

Based on the current CMS schema, the multiple plans that would contain the same EIN and most likely duplicate values within your In-Network Object array. Which if that is the case, then this proposed solution In-Network-Rates File: Schema and Definition of Plan Name and HIOS/EIN #44 would limit it down to a single INN file if approved by CMS.

I do not represent or am affiliated with CMS and your course of action should be vetted with your legal and compliance team for their interpretation of the ruling.
